javascript/funWithFunctions.md

```diff
@@ -0,0 +1,26 @@
+## Functional Expressions in D&D ##
+I've been having fun with functions while creating a JavaScript version of the 5th edition D&D character sheet.
+
+In recreating the character bonus table I got thru 47 lines of inline conditionals before the formula slapped me in the face:  
+![](http://thoughtbot-images.s3.amazonaws.com/TIL/OG_s.jpg)
```

Comment:
Please use the `images.thoughtbot.com` subdomain instead of the bare S3 link so we get Super Fast Loading: http://images.thoughtbot.com/TIL/OG_s.jpg